### DISCUSSION TITLE
Hoe willen we federen implementeren?

### DISCUSSION THREAD

**Discussion by rubenvdlinde** [OpenCatalogi/.github#25] [Q&A] [answered]
Vanuit #13 word gevraag om functionaliteit voor federen, dat roept de vraag op hoe gaan we dit doen?

**Comment by rubenvdlinde** [comment, marked as answer]
- we kiezen voor decentrale data
- zonder duplicatie
- er kan gedeeld worden tussen organisaties
- Er moet een mogelijkheid zijn tot het vaststellen van identiteit van een organisatie. We gaan gewoon een tool kiezen

**Reply by RonaldvCortenberghe** [reply]
Akkoord
